homeassistant/components/switcher_kis/light.py

```diff
@@ -123,3 +115,44 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self._async_call_api(API_SET_LIGHT, DeviceState.OFF, self._light_id)
         self.control_result = False
         self.async_write_ha_state()
+
+
+class SwitcherSingleLightEntity(SwitcherBaseLightEntity):
+    """Representation of a Switcher single light entity."""
+
+    _attr_translation_key = "single_light"
+
+    def __init__(
+        self,
+        coordinator: SwitcherDataUpdateCoordinator,
+        light_id: int = 0,
+    ) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self._light_id = light_id
+        self.control_result: bool | None = None
+
+        # Entity class attributes
+        self._attr_unique_id = f"{coordinator.device_id}-{coordinator.mac_address}"
+
+
+class SwitcherDualLightEntity(SwitcherBaseLightEntity):
+    """Representation of a Switcher dual light entity."""
+
+    _attr_translation_key = "light"
```

Comment:
Please look at the logic used in the cover, you don't need to translate the light for a single entity.

Reply:
It is done, but now the light will get the device name which is not what is shown in the Switcher App. (Just like the cover in the S11 device)

Now it will be like this:
Runner S11:
cover.switcher_runner_6cf5
light.switcher_runner_6cf5_light_1
light.switcher_runner_6cf5_light_2

Runner S12:
cover.switcher_run_plus_a9be_cover_1
cover.switcher_run_plus_a9be_cover_2
light.switcher_run_plus_a9be